active/ZEP0000.md

```diff
@@ -60,29 +64,53 @@ that require collaboration across multiple projects.
 
 - **Specification ZEP**
 
-Specification ZEPs deal with the changes related to [Zarr Specifications].
-The SPEC ZEP (abbreviation of Specification ZEP) for now introduces changes in
-the core specification of Zarr. The changes related to core specification follow a
+Specification ZEPs deal with the changes related to [Zarr Specifications]. The
+SPEC ZEP (abbreviation of Specification ZEP) introduces changes to the
+specification of Zarr. The changes related to specification follow a
 strict and thorough review process and should be adopted by everyone in the
-Zarr community.
+Zarr community. Specification ZEPs are of two types. They are:
 
 - **Core protocol ZEP**
```

Comment:
what is the difference between a specification zep and a core protocol zep?

Reply:
ok I see, a core protocol zep is a subtype of a specification zep. What does the word "protocol" mean here? Are we just using that word to avoid re-using the word "specification"?

Reply:
:+1 to eliminating Protocol and sticking with Specification.